zk_prover/src/chips/overflow/overflow_check.rs

```diff
@@ -6,43 +6,48 @@ use halo2_proofs::plonk::{Advice, Column, ConstraintSystem, Error, Expression, F
 use halo2_proofs::poly::Rotation;
 use std::fmt::Debug;
 
+const MOD_BITS: usize = 252;
```

Comment:
I suggest adding this as generic to the Config too, so one can modify the range check freely

Reply:
Ah, it makes sense. I will fix and revert 16bits test cases.

Reply:
Fixed at [7da5277](https://github.com/summa-dev/summa-solvency/pull/68/commits/7da5277dac72057e94354e80442d60ee1b63bb7a)